build.sbt

```diff
@@ -33,14 +33,18 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
       "org.typelevel" %%% "cats-core" % "2.10.0",
       "org.typelevel" %%% "cats-effect" % "3.5.4",
       "org.scalameta" %%% "munit" % "0.7.29" % Test,
-      "org.typelevel" %%% "munit-cats-effect-3" % "1.0.7" % Test
+      "org.typelevel" %%% "munit-cats-effect-3" % "1.0.7" % Test,
+      "org.scalameta" %%% "munit-scalacheck" % "0.7.29" % Test
     ),
     mimaBinaryIssueFilters ++= Seq(
       ProblemFilters.exclude[ReversedMissingMethodProblem](
         "tech.ant8e.uuid4cats.TimestampedUUIDGeneratorBuilder.tech$ant8e$uuid4cats$TimestampedUUIDGeneratorBuilder$$GeneratorState"
       )
     )
   )
+  .jsSettings(
+    libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.5.0"
```

Comment:
JFYI: There are some folks who are unhappy if this library is included by default – the data in it bloats up their JS artifacts. To avoid it, one can move the `java.time` related methods into a platform specific trait and only expose these on the JVM.

Reply:
This a very valid point. 
Will parametrise the return type and have specialised implementation for each platform.